src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -128,7 +128,7 @@
     <value>Video Conference Mute is a quick and easy way to do a global "mute" of both your microphone and webcam. Disabling this module or closing PowerToys will unmute the microphone and camera.</value>
   </data>
   <data name="VideoConference_CameraAndMicrophoneMuteHotkeyControl_Header.Header" xml:space="preserve">
-    <value>Mute camera &amp; microphone</value>
```

Comment:
Specific reason why moving from '&' to 'and'? Windows 11 Settings seem to favor & over 'and' for most cases (albeit not everywhere).. also more condensed?

Reply:
This is a case of personal opinion; didn't check guidance or something like that. Although it doesn't really matter a ything, as the following images show:
![image](https://user-images.githubusercontent.com/65828559/183284616-4651e473-02b8-442f-b88a-d2cf8a685eb4.png)![image](https://user-images.githubusercontent.com/65828559/183284633-561c9ddc-b011-4487-9c91-620f0e576fcd.png)

Reply:
<img width="272" alt="image" src="https://user-images.githubusercontent.com/9866362/183284923-f9e3fee9-c589-4db9-ab05-68f748a7d336.png">

& seems a bit more formal, and seems to be used in W11 Settings for header for Settings groups (although it's a bit all over the place). Nice Sunday read :): https://medium.com/bpxl-craft/analyzing-the-ampersand-d0dc0daf81f3

Reply:
Nice try. What about https://docs.microsoft.com/style-guide/a-z-word-list-term-collections/a/ampersand ? (MS Docs) 😉

Reply:
I guess you do have a point. That means the display is not how it would be wanted.

Reply:
Maybe I should just use a literal `&` then...